tests/phpunit/tests/privacy/wpPrivacyGeneratePersonalDataExportFile.php

```diff
@@ -654,4 +656,65 @@ public function test_multiple_group_export_single_items_no_group_count_in_toc()
 		$this->assertContains( $request->email, $report_contents );
 
 	}
+
+	/**
+	 * Test should generate a valid JSON report when/if JSON encoding fails.
+	 *
+	 * @ticket 51423
+	 */
+	public function test_should_generate_valid_json_when_json_encoding_fails() {
```

Comment:
Update tests for the latest code change to catch the JSON error and check the message. No longer to open the file and check the JSON.

Reply:
Resolved in commit https://github.com/WordPress/wordpress-develop/pull/1115/commits/368792134756fb448864290a5985643874bf2894